src/Simulation/Simulators/SparseSimulatorDeprecated/SparseSimulatorCS/Probes.cs

```diff
@@ -0,0 +1,255 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
This file can also be deleted - I believe the dump functionality is fully enabled with the code checked in under SparseSimulator.